src/api/app/models/project.rb

```diff
@@ -1815,6 +1815,20 @@ def to_param
     name
   end
 
+  def maintainers
```

Comment:
Just a thought... it can be improved with chained scopes in the Relationship model?